File: CS/1018 - Cedulas.cs
using System;

class URI
{
	static void Main()
	{
		int resto;
		
		int money = int.Parse(Console.ReadLine());

		int cem = money / 100;
		resto = money % 100;

		int cinq = resto / 50;
		resto = resto % 50;

		int vinte = resto / 20;
		resto = resto % 20;

		int dez = resto / 10;
		resto = resto % 10;

		int cinco = resto / 5;
		resto = resto % 5;

		int dois = resto / 2;
		resto = resto % 2;

		Console.WriteLine(money);

		Show_Info(cem, 100);
		Show_Info(cinq, 50);
		Show_Info(vinte, 20);
		Show_Info(dez, 10);
		Show_Info(cinco, 5);
		Show_Info(dois, 2);
		Show_Info(resto, 1);
	}

	public static void Show_Info(int valor, int nota)
	{
		Console.Write("{0} nota(s) de R$ " + nota + ",00\n", valor);
	}
}

File: PHP/1018 - Cedulas.php
<?php
	
	function show_info($valor, $nota){

		echo "$valor nota(s) de R$ $nota,00\n";
	}	

	$valor = (int)readline();
	echo "$valor\n";

	$cem = (int)($valor / 100);
	$resto = $valor % 100;

	$cinq = (int)($resto / 50);
	$resto = $resto % 50;

	$vinte = (int)($resto / 20);
	$resto = $resto % 20;

	$dez = (int)($resto / 10);
	$resto = $resto % 10;

	$cinco = (int)($resto / 5);
	$resto = $resto % 5;

	$dois = (int)($resto / 2);

	$um = $resto % 2;

	show_info($cem, 100);
	show_info($cinq, 50);
	show_info($vinte, 20);
	show_info($dez, 10);
	show_info($cinco, 5);
	show_info($dois, 2);
	show_info($um, 1);

?>